{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -16,6 +16,8 @@ FROM inveniosoftware/centos8-python:3.7
 FROM inveniosoftware/centos8-python:3.8
 {%- endif %}
 
+RUN yum install -y ImageMagick
```

Comment:
Shouldn't this go in ``inveniosoftware/centos8-python``?

Reply:
I thought that a base invenio image might or might not use iiif... right?

Reply:
Yes, but it's like dejavu font and lib cairo.....instances might  or might not use the badge feature, but we include in the base image so that it has the basics dependencies

Reply:
Okay creating a task then to move it https://github.com/inveniosoftware/cookiecutter-invenio-rdm/issues/162